README.md

```diff
@@ -39,37 +40,36 @@ The openframeworks forum:
 
 [http://forum.openframeworks.cc/](http://forum.openframeworks.cc/)
 
-is a warm and friendly place.  Please ask or answer a question.  The most important part of this project is that it's a community, more than just a tool, so please join us!  Also, this is free software, and we learn so much about what is hard, what doesn't make sense, what is useful, etc. The most basic questions are acceptable here!  Don't worry, just join the conversation.  Learning in OF is social, it's hard to do it alone, but together we can get far!
```

Comment:
proposed change affects tone of voice - its nicer to say please, and underlines the argument that OF is a warm and friendly place =)